iidm/iidm-extensions/src/main/java/com/powsybl/iidm/network/extensions/TwoWindingsTransformerToBeEstimated.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network.extensions;
+
+import com.powsybl.commons.extensions.Extension;
+import com.powsybl.iidm.network.TwoWindingsTransformer;
+
+import java.util.Set;
+
+/**
+ * Indicate for a state estimation which tap changers are to be estimated (i.e. their tap positions should be outputs).
+ * If a tap changer is not to be estimated, it should not be changed during a state estimation (i.e its tap position is only an input).
+ *
+ * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
+ */
+public interface TwoWindingsTransformerToBeEstimated extends Extension<TwoWindingsTransformer> {
+
+    /**
+     * A two windings transformer can have a ratio tap changer and/or a phase tap changer that can be estimated during a state estimation.
+     */
+    enum TapChanger {
+        RATIO_TAP_CHANGER,
+        PHASE_TAP_CHANGER
+    }
+
+    @Override
+    default String getName() {
+        return "twoWindingsTransformerToBeEstimated";
+    }
+
+    /**
+     * Get tap changers that are to be estimated during a state estimation.
+     */
+    Set<TapChanger> getTapChangers();
```

Comment:
No strong opinion here, but I wonder if it would not have been more natural with plain booleans (2 here and 6 for the 3 windings).

No problem to keep it this way if it's OK for everyone.

Reply:
I actually agree with you, I think it would be more simple with booleans...